tools/dotnet-deb-tool/ArgumentEscaper.cs

```diff
@@ -0,0 +1,206 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Dotnet.Deb.Tool
+{
+    public static class ArgumentEscaper
```

Comment:
Is this an exact dupe of https://github.com/dotnet/core-setup/blob/master/build_projects/Microsoft.DotNet.Cli.Build.Framework/ArgumentEscaper.cs?  If so, maybe we can just compile it into the deb-tool using shared sources?  That way it isn't duplicated for a third time (in case it ever needs to change).